includes/StringUtils.h

```diff
@@ -0,0 +1,16 @@
+#ifndef STRING_UTILS_H_DEFINED
+#define STRING_UTILS_H_DEFINED
+#include <vector>
+#include <string>
+
+
+void readMapFile(const char *filename, std::vector<std::string>&ret);
+
+std::vector<std::string> splitString(std::string original, char ch);
+std::vector<char> splitChar(std::string original, std::string ignoredChars);
```

Comment:
```suggestion
std::vector<char> splitChar(const std::string& original, const std::string& ignoredChars);
```
You should rethink this, because you'll be returning a temp vector. You know what happens when a vector define on a function, right?